webrtc.html

```diff
@@ -6640,6 +6640,7 @@ <h3><dfn>RTCRtpCodecCapability</dfn> Dictionary</h3>
         <pre class="idl">dictionary RTCRtpCodecCapability {
              required DOMString mimeType;
              required unsigned long  clockRate;
+             required octet preferredPayloadType;
```

Comment:
I think this only solves the problem if you are the offerer. If you are the answerer, it is the other endpoint that announced the codec payload types, right?